Dockerfile

```
@@ -0,0 +1,48 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.2
+
+ARG VCS_REF
+ARG VCS_URL
+ARG IMAGE_NAME
+ARG IMAGE_DESCRIPTION
+ARG IMAGE_DISPLAY_NAME
+ARG IMAGE_NAME_ARCH
+ARG IMAGE_MAINTAINER
+ARG IMAGE_VENDOR
+ARG IMAGE_VERSION
+ARG IMAGE_RELEASE
+ARG IMAGE_SUMMARY
+ARG IMAGE_OPENSHIFT_TAGS
+
+LABEL org.label-schema.vendor="Red Hat" \
+      org.label-schema.name="$IMAGE_NAME_ARCH" \
+      org.label-schema.description="$IMAGE_DESCRIPTION" \
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.vcs-url=$VCS_URL \
+      org.label-schema.license="Red Hat Advanced Cluster Management for Kubernetes EULA" \
+      org.label-schema.schema-version="1.0" \
+      name="$IMAGE_NAME" \
+      maintainer="$IMAGE_MAINTAINER" \
+      vendor="$IMAGE_VENDOR" \
+      version="$IMAGE_VERSION" \
+      release="$IMAGE_RELEASE" \
+      description="$IMAGE_DESCRIPTION" \
+      summary="$IMAGE_SUMMARY" \
+      io.k8s.display-name="$IMAGE_DISPLAY_NAME" \
+      io.k8s.description="$IMAGE_DESCRIPTION" \
+      io.openshift.tags="$IMAGE_OPENSHIFT_TAGS"
+
+RUN microdnf install ca-certificates vi --nodocs &&\
+    mkdir /licenses &&\
+    microdnf clean all
+
+ENV VCS_REF="$VCS_REF" \
+    USER_UID=1001
+
+
+
+ADD output/insights-client /bin
+
+
+EXPOSE 3010
```

Should this be `3030` to match default?
```suggestion
EXPOSE 3030
```

---

Fixed it